gorp.go

```diff
@@ -764,7 +764,20 @@ func (m *DbMap) readStructColumns(t reflect.Type) (cols []*ColumnMap) {
 				}
 			}
 		} else {
-			columnName := f.Tag.Get("db")
+			// Tag = Name { ','  Option }
+			// Option = OptionKey [ ':' OptionValue ]
+			cArguments := strings.Split(f.Tag.Get("db"), ",")
+			columnName := cArguments[0]
+			var maxSize int
+			for _, argString := range cArguments[1:] {
+				arg := strings.SplitN(argString, ":", 2)
+				switch arg[0] {
+				case "size":
+					maxSize, _ = strconv.Atoi(arg[1])
+				default:
+					//log.Printf("Unrecognized argument: %v\n", arg)
```

Comment:
I do think we should error out, here.  If code expects an option to exist, we shouldn't just ignore it.